examples/basic.yml

```
@@ -0,0 +1,10 @@
+---
+- name: Simple deploy PostgreSQL server version 10
+  hosts: localhost
+  vars:
+    postgresql_version: "10"
+
```

```suggestion
  roles:
    - linux-system-roles.postgresql
```
In this case, you can simplify the playbook by `roles` within the head of the role, tasks section then not needed.

---

I see you do this in other examples, please fix here too